### DISCUSSION TITLE
How to clone site

### DISCUSSION THREAD

**Discussion by sicom-dn** [EasyEngine/easyengine#1664] [Support Requests]
So i need to make a staging copy of working site. As i understand this can be done with clone command. But when i run
`ee site clone domain.com staging.domain.com` i get the error `Error: Error while executing clone-create-site. closure: Unable to find 'staging.domain.com' on 'localhost'`. What i do wrong?

**Comment by scollonp** [comment]
Only works on WordPress sites currently. Is the source site a WP site?

**Reply by sicom-dn** [reply]
Yes, WP

**Reply by marcoandrei-waycomm** [reply]
But did you create the site with --wp option?

**Comment by tosinamuda** [comment]
@scollonp did you get any fix?

**Reply by scollonp** [reply]
I'm not looking for a solution @tosinamuda, but see @WojciechRhdt's reply below.

**Comment by WojciechRhdt** [comment]
I have the same problem. Any suggestions how to copy a WP-Site in EE?

Edit: I had a DNS issue with the domain. After fixing this, I was able to clone a site with the command ee clone site old.domain.com new.domain.com --ssl=le

**Reply by techjewel** [reply]
@WojciechRhdt What was the DNS issue? Would you please share details? I am getting the same errors.

**Reply by rickdroio** [reply]
same issue here, any updates?

**Comment by dinhngocdung** [comment]
get more infomation to understand : ee site clone domain.com staging.domain.com --debug.
I saw my problem is ssl, and I change comannd: ee site clone domain.com staging.domain.com --ssl=off --debug
I update ssl for staging.domain.com later with ee site update

**Reply by marcoandrei-waycomm** [reply]
Hello, @dinhngocdung! It appears that the problem was resolved, right? Good!

Maybe we can mark this discussion as "answered"?

Cheers! Saúde!
